### DISCUSSION TITLE
Selección de tema para proyecto del curso

### DISCUSSION THREAD

**Discussion by elverarroyave-udea** [rramosp/ai4eng.v1#143] [00 - General]
Buenas noches profesor, esperamos se encuentre bien.

A continuación listamos los temas que hemos visto interesante y que creemos adaptables al proyecto del curso.

1. Predict Future Sales  - enlace ---> [aquí ](https://www.kaggle.com/competitions/competitive-data-science-predict-future-sales/data?select=item_categories.csv)
2. Store Sales - Time Series Forecasting  - enlace ---> [aquí](https://www.kaggle.com/competitions/store-sales-time-series-forecasting)
3. Prediccion Titanic Regresion Logistica - enlace ---> [aquí](https://www.kaggle.com/code/alexbonella/prediccion-titanic-regresion-logistica/notebook)

Esperamos por favor nos pueda ayudar con sus comentarios para la elección de uno de ellos. 

-Saludos.

**Comment by Felipe-RA** [comment]
Saludos elver

La predicción del titanic no es válida porque la trabajamos en el curso

El de time series pinta bastante bien!
